data/MI/incentives.json

```diff
@@ -0,0 +1,1400 @@
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $500 rebate on Installation Costs for a Qualified Level 2 Charger."
+    }
+  },
+  {
+    "id": "MI-21",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 850
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$850 for 19+ SEER or 18+ SEER2 Air Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-22",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 750
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$750 for 18 SEER or 17-17.99 SEER2 Air Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-23",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$250 for 16-17 SEER or 15.2-16.99 SEER2 Air Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-24",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 150
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$150 for 15 SEER or 14.3-15.19 SEER2 Air Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-25",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 800
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$800 for 17+ SEER or 16+ EER2 Ground Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-26",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$1000 for 18+ SEER and HSPF 8.8+  OR  17+ SEER2 and HSPF2 7.5+."
```

Comment:
I'd go with `18+ SEER and HSPF 8.8+, or 17+ SEER2 and HSPF2 7.5+`. The all-caps OR is a little odd.